net-misc/seafile/seafile-8.0.6.ebuild

```
@@ -0,0 +1,58 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..10} )
+
+RELEASE_COMMIT="7a61feaba63006d11c09d1020593ba3fcb3fc717"
+
+inherit autotools python-single-r1 vala
+
+DESCRIPTION="File syncing and sharing software with file encryption and group sharing"
+HOMEPAGE="https://www.seafile.com/ https://github.com/haiwen/seafile/"
+SRC_URI="https://github.com/haiwen/${PN}/archive/${RELEASE_COMMIT}.tar.gz -> ${P}.tar.gz"
```

To save a bit of manual work each update, you can use `${PV}` here like this:

```suggestion
SRC_URI="https://github.com/haiwen/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

(`S="..."` will also need changing)

---

Hi,
I was using commit hash since some versions ago because upstream was changing tags to other commit without warning.
Then I thought it was better to keep this behavior to avoid unnecessary issues.

---

Yes, seconding this. I've seen this upstream moving tags around before, and not only once.

fwiw maybe a note in the ebuild be nice so other developers don't wonder why every time?